### PR TITLE
NugetNugget Tidyup

### PR DESCRIPTION
**Pull request by DavidChristiansen**
- addition of modernizr to project
- css update

### CODE REVIEW

**Comment by davidfowl** [comment]
You need to merge with upstream.

**Comment by DavidChristiansen** [comment]
Not sure I did that right.

File: JabbR/Chat.nuget.css [hunk 1]
```diff
@@ -87,12 +87,23 @@
...
 .nuget-projecticon {
-    float: left;
+    float: right;
     height: 32px;
-    margin: 10px;
-    width: 32px;
+    margin-left: 10px;
+    margin-bottom: 10px;
+    width: 32px;
+	padding: 10px;
```
Comment (davidfowl) [inline JabbR/Chat.nuget.css]: formatting.

Comment (DavidChristiansen) [inline JabbR/Chat.nuget.css]: Good eye

**Review by Scutua — APPROVED** [review]

**Pull request merged**